src/scripts/main.js

```diff
@@ -1,5 +1,60 @@
+/* eslint-disable no-undef */
+// they sad "getComputedStyle is not undefined"
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const spider = document.querySelector('.spider');
+const spiderWidth = parseFloat(getComputedStyle(spider).width);
+const spiderHeight = parseFloat(getComputedStyle(spider).height);
```

Comment:
spider has equal sides, you don't need 2 separate variables
also, it's a bad practice to take element size from CSS -> [article](https://learn.javascript.ru/size-and-scroll#ne-stoit-brat-width-height-iz-css)

Reply:
I I was counting on the opportunity of different sides, but ok